src/item.cpp

```diff
@@ -218,6 +219,22 @@ item& item::deactivate( const Character *ch, bool alert )
     return *this;
 }
 
+item& item::activate()
+{
+    if( active ) {
+        return *this; // no-op
```

Comment:
For consistency with the related classes. There are examples of chaining in `iuse.cpp`